### PR TITLE
Corrected app info due to changed functionality

### PR DESCRIPTION
**Pull request by RadekRo**

### CODE REVIEW

**Review by KamilaKisiel — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -55,8 +55,8 @@ <h3>&#x21E1;</h3>
     <div class="header-text-container">
         <h1>WYSZUKIWARKA</h1>
         <h2 class="shadow">CZĘŚCI SAMOCHODOWYCH</h2>
-        <h3 class="shadow">Pomożemy Ci znaleźć <a class="marker">każdą część</a> do Twojego samochodu,<br/>a jako <a class="marker">BONUS</a> wskażemy Ci najbliższy
-        warsztat!</h3>
+        <h3 class="shadow">Pomożemy Ci znaleźć <a class="marker">każdą część</a> do Twojego samochodu,<br/>a jako <a class="marker">BONUS</a> zachowamy ją dla
+             Ciebie w Ulubionych!</h3>
         <h3 class="shadow">Startujemy już wkrótce!</h3>
```
Comment (JarekSopinski) [inline index.html:60]: "a jako BONUS będziesz mógł zachować ją w ulubionych" ?
It's user who saves parts for himself, not app developers :D

Comment (RadekRo) [reply index.html:60]: I don't understand your comment. Please make it clear .

Comment (JarekSopinski) [reply index.html:60]: I mean that when user saves part, he does it by himself - it's not done by some other people. It sounded a bit strange to me ;) It's as if online shop had a line like this: "We will buy stuff for you", not "You can buy stuff" ;) Therefore I proposed changing this line to ""a jako BONUS będziesz mógł zachować ją w ulubionych".

Comment (RadekRo) [reply index.html:60]: Pomożemy Ci znaleźć każdą część do Twojego samochodu, a jako BONUS zachowamy ją dla Ciebie w Ulubionych! - application saves the part for user thanks to the login. It also quite understandable.

Comment (RadekRo) [reply index.html:60]: I don't think it is wrongly written. Kamila also approved this part so I guess it might stay as it is.

Comment (KamilaKisiel) [reply index.html:60]: Yep, looks good to me

Comment (JarekSopinski) [reply index.html:60]: Ok, I have other opinion but I won't argue about that :)

**Pull request merged**